Automattic/wp-calypso
# Hosting: Show success notification after Atomic transfer

Fixes https://github.com/Automattic/dotcom-forge/issues/9212

## Proposed Changes

Displays a success notification after activating the hosting features:

<img width="1330" alt="Screenshot 2024-09-26 at 12 29 42" src="https://github.com/user-attachments/assets/fc584d23-5061-42d7-8997-4e89e32d4819">


## Why are these changes being made?

To inform the user that their hosting features have been activated.



## Testing Instructions

- Use the Calypso live link below
- Go to `/sites`
- Select a Simple site with a Business plan
- Open the "Hosting Features" tab
- Activate them
- Wait for the Atomic transfer to finish
- Make sure a success notification shows up

## Pre-merge Checklist
